apps/codeforafrica/src/pages/about/members/[slug].page.js

```diff
@@ -21,32 +15,60 @@ function Index({ member, sections, ...props }) {
       <Section
         sx={{
           px: { xs: 2.5, sm: 0 },
+          pt: { xs: 2.5, md: 7 },
           maxWidth: {
             sm: "648px",
             md: "912px",
           },
         }}
       >
-        <Box
+        <RichTypography
+          variant="body1"
           sx={{
-            color: "grey.main",
-            rowGap: 2,
-            display: "flex",
-            flexDirection: "column",
+            mb: 5,
+            typography: "subheading",
           }}
         >
-          <Typography variant="footerCap">Connect</Typography>
-          <ShareBar>
-            <FacebookShareBarButton />
-            <LinkedinShareBarButton />
-            <TwitterShareBarButton />
-          </ShareBar>
-        </Box>
+          {member?.description}
+        </RichTypography>
+        <ShareThisPage
+          spacing="17px"
+          title="Connect"
+          sx={{
+            color: "text.primary",
+          }}
+        />
       </Section>
       {sections?.map((section) => {
         switch (section.slug) {
           case "related-projects":
-            return <RelatedProjects {...section} key={section.slug} />;
+            return (
+              <React.Fragment key={section.slug}>
+                <SectionDivider
+                  sx={{
+                    maxWidth: {
+                      sm: "648px",
```

Comment:
The values of most of these viewports seem the same; Couldn't we have them in a 'global' location?

Reply:
Ideally [theme](https://github.com/CodeForAfrica/ui/blob/14638e687cf125cff0332ba461939490cc2fb19e/apps/codeforafrica/src/theme/index.js#L32-L40) is where they would have been but some pages like this one uses different sizes... If you check [Figma](https://www.figma.com/file/MbCygAaNhIjgJOHrQZpZ2U/CfA---CfA-site?node-id=7255%3A41478), you can see even the above is just estimation i.e. it changes from one section to the next. I think we'd need to confirm with the designer first as part of the review.